src/pages/Pokemon.js

```diff
@@ -1,17 +1,18 @@
 import { useEffect, useState } from "react";
+import { useRef } from "react/cjs/react.development";
 import "./Pokemon.css";
 
 export default function Pokemon({ Link }) {
   const [pagecount, setPagecount] = useState(0);
   const [pokemonlist, setPokemonlist] = useState([]);
-  const [nextpage, setNextpage] = useState("https://pokeapi.co/api/v2/pokemon");
+  const nextpage = useRef("https://pokeapi.co/api/v2/pokemon");
```

Comment:
the only thing is changing in the next and previous api is the offset number. "https://pokeapi.co/api/v2/pokemon?offset=80&limit=20"

maybe you can make use of that in order to have just one const (not next page, but just page, for next and previous)